packages/in_app_purchase/README.md

```diff
@@ -0,0 +1,82 @@
+
+## Supported devices
+
+This plugin is only supported on Smart TVs running Tizen 4.0 and above.
+
+## Required privileges
+
+To use this plugin in a Tizen application, you need to declare the following privileges in your `tizen-manifest.xml` file.
+
+```xml
+<privileges>
+  <privilege>http://developer.samsung.com/privilege/billing</privilege>
+  <privilege>http://developer.samsung.com/privilege/sso.partner</privilege>
+  <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+</privileges>
+```
+
+The sso.partner privilege is required by the [Sso API](https://developer.samsung.com/smarttv/develop/api-references/samsung-product-api-references/sso-api.html) to internally obtain the user's custom ID (UID). Your app must be signed with a [partner-level certificate](https://docs.tizen.org/application/dotnet/get-started/certificates/creating-certificates) to use this privilege.
+
+## Preparation
+
+Follow these steps before setting up in-app purchases for your application:
+
+1. [Register your application](https://github.com/flutter-tizen/flutter-tizen/blob/master/doc/publish-app.md) at the [Samsung Apps TV Seller Office](https://seller.samsungapps.com/tv) if you haven't registered yet. You do not need to complete the registration process at this point. Go to the **Billing Info** page of the app and set the **Samsung Checkout** checkbox to ON. You can return back to this page and finish the registration process when the final version of your app is ready.
+
+2. Log in to the [Samsung Checkout DPI Portal](https://dpi.samsungcheckout.com) and register your in-app items. You can find your **App ID** and **Security Key** in the [**App Details Setting**](https://dpi.samsungcheckout.com/settings/appdetails) page. These values will be used as request parameters in your app code.
+
+## Usage
+
+This package is not an _endorsed_ implementation of `in_app_purchase`. Therefore, you have to include `in_app_purchase_tizen` alongside `in_app_purchase` as dependencies in your `pubspec.yaml` file.
+
+```yaml
+dependencies:
+  in_app_purchase: ^3.1.4
+  in_app_purchase_tizen: ^0.1.0
+```
+
+Then you can import `in_app_purchase` and `in_app_purchase_tizen` in your Dart code:
+
+```dart
+import 'package:in_app_purchase/in_app_purchase.dart';
+import 'package:in_app_purchase_tizen/in_app_purchase_tizen.dart';
+```
+
+You must call `setRequestParameters` to set required parameters before making any plugin API call.
+You can find your `countryCode` in [Samsung Developers: Implementing the Purchase Process](https://developer.samsung.com/smarttv/develop/guides/samsung-checkout/implementing-the-purchase-process.html#Country-and-Currency-Codes) and `serverType` in [Samsung Developers: Billing API References](https://developer.samsung.com/smarttv/develop/api-references/samsung-product-api-references/billing-api.html#1.-Type-Definitions)
```

Comment:
Is the country and currency code the user's or the app developer's? What if I added multiple countries/regions in the product detail settings?

Also how can I determine the server type (how should I configure my product for each server in the DPI portal)?

Reply:
I noticed this information in the Billing Info page of the Seller Office:

> The app must check checkout-available country/location before calling the billing API. Ensure that the 'Billing Service Available Country/location Check API’ is integrated to your app to check checkout-available country/location.

> Please ensure that Checkout is supported in your service country/location by using the Billing Service Available Country/location Check API. Once confirmed, your application can call the billing API at [Checking Billing Service Available Country/Location.](https://developer.samsung.com/smarttv/develop/guides/samsung-checkout/implementing-the-purchase-process.html#Checking-Billing-Service-Available-Country%2FLocation)

so "your service country" probably refers to where the end user (consumer) lives. How do I know where the user's region is? Also do we have an API for checking the service availability for the region in Dart?

Reply:
> so "your service country" probably refers to where the end user (consumer) lives. How do I know where the user's region is?
As I can see the [prerequisites ](https://developer.samsung.com/smarttv/develop/guides/samsung-checkout/implementing-the-purchase-process.html#Prerequisites),  the countryCode is (user's) TV countryCode. 
Web use `getSystemConfig` to get the countryCode.

> Also do we have an API for checking the service availability for the region in Dart?
I will find a capi of checking service avaiable country/location.